.gitattributes

```diff
@@ -1,2 +1,5 @@
 *.zip filter=lfs diff=lfs merge=lfs -text
 *.png filter=lfs diff=lfs merge=lfs -text
+*.svg filter=lfs diff=lfs merge=lfs -text
```

Comment:
I don't think we need LFS for SVG, I'm not against it, but maybe if there are bitmaps embedded directly into the file then this makes sense.

Reply:
:+1:  @ekarademir  - I didn't fully read the docs. I'll push an update

Reply:
Don't need to spend too much time on this @codecrunchers you can merge it as is.